sim/warlock/lifetap.go

```diff
@@ -46,6 +47,14 @@ func (warlock *Warlock) registerLifeTapSpell() {
 			if hasGlyph {
 				warlock.GlyphOfLifeTapAura.Activate(sim)
 			}
+
+			if has4PT7 {
+				if warlock.SpiritsoftheDamnedAura.IsActive() {
```

Comment:
`aura.Activate()` already checks for `IsActive()` and does the refresh. You can simplify this code here, which will also cause the PPM metrics for the aura to be correct.

Reply:
Oh, nice.